src/uu/expr/src/expr.rs

```diff
@@ -94,16 +94,31 @@ pub fn uu_app() -> Command {
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     // For expr utility we do not want getopts.
     // The following usage should work without escaping hyphens: `expr -15 = 1 +  2 \* \( 3 - -4 \)`
-    let matches = uu_app().try_get_matches_from(args)?;
-    let token_strings: Vec<&str> = matches
-        .get_many::<String>(options::EXPRESSION)
-        .map(|v| v.into_iter().map(|s| s.as_ref()).collect::<Vec<_>>())
-        .unwrap_or_default();
+    let args: Vec<String> = args
+        .skip(1) // Skip binary name
+        .map(|a| a.to_string_lossy().to_string())
+        .collect();
 
-    let res: String = AstNode::parse(&token_strings)?.eval()?.eval_as_string();
-    println!("{res}");
-    if !is_truthy(&res.into()) {
-        return Err(1.into());
+    if args.len() == 1 && args[0] == "--help" {
+        let help_msg: String = format!(
```

Comment:
You can still use clap to print this information if you want with something like `uu_app().print_help()` (I forgot the exact function)